examples/pytorch/BertNewsClassification/bert_classification.py

```diff
@@ -252,8 +252,8 @@ def forward(self, input_ids, attention_mask):
 
         :return: output - Type of news for the given news snippet
         """
-        _, pooled_output = self.bert_model(input_ids=input_ids, attention_mask=attention_mask)
-        output = F.relu(self.fc1(pooled_output))
+        output = self.bert_model(input_ids=input_ids, attention_mask=attention_mask)
+        output = F.relu(self.fc1(output.pooler_output))
```

Comment:
In transformers < 4.0.0, `self.bert_model` returns a tuple (of tensors) but in >= 4.0.0 it returns a dict-like object and
a dict key (str) is assigned to `pooled_output` which must be a torch tensor.
```python
# unpack dict

>>> _, b = {"a": 0, "b": 1}
>>> b
'b'
```